app/javascript/components/pages/Services.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react'
+
+export default class Services extends Component {
+  render() {
+    return (
+        <>
+            <h1>Services</h1>
+        <div>
+                <h3>The Tribe</h3>
+                <h5>https://support.therapytribe.com/</h5>
+                <p>At TherapyTribe,they believe that individuals become empowered to help themselves and others when they feel a part of something larger. Their free online support community offers members a convenient and safe place to connect. They’ve combined members from five long-standing support websites to create one fantastic wellness community.</p>
+        </div>
+        <div>
+                <h3>Nami
+(National alliance in mental health)</h3>
+                <h5>https://www.nami.org/Home</h5>
+                <p> NAMI, the National Alliance on Mental Illness, is the nation’s largest grassroots mental health organization dedicated to building better lives for the millions of Americans affected by mental illness.</p>
+        </div>
+        <div>
+                <h3>Better Help</h3>
+                <h5>https://www.betterhelp.com/</h5>
+                <p>Makes professional therapy accessible, affordable, and convenient — so anyone who struggles with life’s challenges can get help, anytime and anywhere. </p>
+        </div>
+        <div>
+            <h3>PostpartumProgress</h3>
+            <h5>https://postpartumprogress.com/about</h5>
+            <p>Postpartum is the world’s most widely-read blog dedicated to maternal mental illness.</p>
+        </div>
+         <div>
+            <h3>DBSA(Depression and Bipolar Support Alliance)</h3>
+            <h5>https://www.dbsalliance.org/education/</h5>
+            <p>DBSA provides hope, help, support, and education to improve the lives of people who have mood disorders.</p>
+        </div>
+        <div>
+            <h3>988Suicide&crisisLifeline</h3>
+            <h5>https://988lifeline.org/</h5>
+            <p>The Lifeline provides 24/7, free and confidential support for people in distress, prevention and crisis resources for you or your loved ones, and best practices for professionals in the United States.</p>
+         </div>
+         <div>
+            <h3>Healthunlocked</h3>
+            <h5>https://healthunlocked.com/</h5>
+            <p>Imagine a world where you can tap into the health experiences of millions of people – and translate these into an action plan with the support of peers and patient organisations.  </p>
+         </div>
+         <h3>Zencare</h3>
+            <h5>https://zencare.co/</h5>
+            <p>Zencare started as a solution to a common problem: it's difficult to find a great-fitting therapist through an online search.Founder Yuri Tomikawa began Zencare after spending weeks looking for the right therapist. Our goal is to remove the guesswork and logistical hassle from the therapist search, providing a smooth and empowering experience.</p>
+            </>
```

Comment:
Run a code formatter over this and push up again to the same branch.

Reply:
I just pushed it again and added links